aten/src/ATen/native/native_functions.yaml

```diff
@@ -5767,13 +5767,15 @@
   dispatch:
     CPU: adaptive_avg_pool3d_out_cpu
     CUDA: adaptive_avg_pool3d_out_cuda
+    QuantizedCPU: quantized_adaptive_avg_pool3d_out
 
 - func: adaptive_avg_pool3d(Tensor self, int[3] output_size) -> Tensor
   use_c10_dispatcher: full
   python_module: nn
   dispatch:
     CPU: adaptive_avg_pool3d_cpu
     CUDA: adaptive_avg_pool3d_cuda
+    QuantizedCPU: quantized_adaptive_avg_pool3d
```

Comment:
I feel we should actually rename this, it should be `adaptive_avg_pool3d_quantized_cpu` same for others

Reply:
I think I can do it in a separate PR -- it involves changing other ones. WIll creaete a task for myself

Reply:
yeah we don't need to it in this PR, sounds good, thanks

Reply:
#40315

Reply:
#41071